components/TextControls/examples/MDCTextControlContentViewController.h

```diff
@@ -0,0 +1,103 @@
+// Copyright 2019-present the Material Components for iOS authors. All Rights Reserved.
```

Comment:
Gentle reminder to move this to the `supplemental` directory to clarify that it's not an example.